scripts/generate_rust_target.rs

```diff
@@ -227,7 +227,6 @@ fn main() {
     ts.push("emit-debug-gdb-scripts", false);
     ts.push("frame-pointer", "may-omit");
     ts.push("function-sections", false);
-    ts.push("position-independent-executables", true);
```

Comment:
While this isn't necessary here I believe the commut message is incorrect. I think it can also affect the relocation model that is used when using the bin crate type.

Reply:
I may be missing something, but I only saw it used in `compiler/rustc_codegen_ssa/src/back/link.rs`'s `link_output_kind()`.

Reply:
Just checked it and you're right.

Reply:
Thanks for double-checking!